### DISCUSSION TITLE
nuxt will remove <script type="text/plain"> but I need it for AMP website

### DISCUSSION THREAD

**Discussion by ahmad-666** [nuxt/nuxt#10761] [Questions (Nuxt 2)]
Hi 
nuxt will remove any inline <script type="text/plain">from template for security reasons but I want to write custom js code for AMP website ... how can I achieve this?
I am using NuxtV2

**Comment by ghost** [comment]
I think you can use the vue v-html directive if you really need to add something like that. It accepts a string value, which will be added as raw html.
https://vuejs.org/guide/essentials/template-syntax.html
